### PR TITLE
Use the naming convention OBS expects for ISOs

### PR DESCRIPTION
**Pull request by davidcassany**

### CODE REVIEW

**Comment by kkaempf** [comment]
There's a naming convention ? 😮  Where's this documented ? 😕

**Comment by davidcassany** [comment]
> There's a naming convention ? 😮 Where's this documented ? 😕

I don't know of any documentation, just empiric knowledge, probably there are some docs, but I doubt it. I am afraid the only _documentation_ might be OBS and KIWI code 😓 I think OBS just expects KIWI naming or vice versa.

I could confirm this fixes static links [here](https://download.opensuse.org/repositories/home:/dcassany:/branches:/isv:/Rancher:/Elemental:/Staging/containers/iso/), see the downloadable image `sl-micro-6.0-baremetal.x86_64.iso`. These one does not exist in current staging...

File: .obs/dockerfile/micro-baremetal-iso/Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ RUN zypper --installroot /busybox in --no-recommends -y busybox
 
 # Version value is taken form the elemental repository tags
 # Release value of this image and os image are unrelated
-RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-baremetal-$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
+RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-baremetal.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```
Comment (kkaempf) [inline .obs/dockerfile/micro-baremetal-iso/Dockerfile:27]:
```suggestion
# must be ...<flavor>.<arch>... for OBS
RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-baremetal.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```

File: .obs/dockerfile/micro-base-iso/Dockerfile [hunk 2]
```diff
@@ -23,7 +23,7 @@ RUN zypper --installroot /busybox in --no-recommends -y busybox
 
 # Version value is taken form the elemental repository tags
 # Release value of this image and os image are unrelated
-RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-base-$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
+RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-base.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```
Comment (kkaempf) [inline .obs/dockerfile/micro-base-iso/Dockerfile:27]:
```suggestion
# must be ...<flavor>.<arch>... for OBS
RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-base.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```

File: .obs/dockerfile/micro-kvm-iso/Dockerfile [hunk 3]
```diff
@@ -23,7 +23,7 @@ RUN zypper --installroot /busybox in --no-recommends -y busybox
 
 # Version value is taken form the elemental repository tags
 # Release value of this image and os image are unrelated
-RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-kvm-$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
+RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-kvm.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```
Comment (kkaempf) [inline .obs/dockerfile/micro-kvm-iso/Dockerfile:27]:
```suggestion
# must be ...<flavor>.<arch>... for OBS
RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-kvm.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```

File: .obs/dockerfile/micro-rt-iso/Dockerfile [hunk 4]
```diff
@@ -24,7 +24,7 @@ RUN zypper --installroot /busybox in --no-recommends -y busybox
 
 # Version value is taken form the elemental repository tags
 # Release value of this image and os image are unrelated
-RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-rt-$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
+RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-rt.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```
Comment (kkaempf) [inline .obs/dockerfile/micro-rt-iso/Dockerfile:28]:
```suggestion
# must be ...<flavor>.<arch>... for OBS
RUN elemental --debug --config-dir . build-iso -o /output -n "sl-micro-%%SLMICRO_VERSION%%-rt.$(uname -m)-%VERSION%-Build%RELEASE%" dir:rootfs
```

**Review by kkaempf — CHANGES_REQUESTED** [review]
we should add a bit of documentation for future generations 😆

**Review by kkaempf — APPROVED** [review]
🤩

**Pull request merged**